Common Distributions-Distributions.ipynb

```diff
@@ -0,0 +1,930 @@
+{
```

Comment:
<p>I would prefer if we used real-world examples everywhere, that's our differentiator. How about which party a random person from a country would vote for?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Way better Thanks</p><p> </p>